Sync `mediaqueries` from WPT upstream

iidmsa opened the pull request:
#### cf2c6d9f667cbaf1547d0d245a56c9e9cbd32ae7
<pre>
Sync `mediaqueries` from WPT upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=294362">https://bugs.webkit.org/show_bug.cgi?id=294362</a>
<a href="https://rdar.apple.com/problem/153140956">rdar://problem/153140956</a>

Reviewed by Anne van Kesteren.

Upstream commit: web-platform-tests/wpt@aae479b

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/reference/ref-green-body.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/296151@main">https://commits.webkit.org/296151@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d02b51a2b0d8be8710479ede99272dab5b3638f9

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/107470 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/27152 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/17560 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/112684 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/58006 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/27836 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/35652 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/81578 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/110399 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/22043 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/96856 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/61963 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/21480 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/14997 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/57450 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/91410 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/15028 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/115785 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/34537 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/25448 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/90616 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/34912 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/93112 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/90362 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23054 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/35273 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/13049 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/30282 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/34458 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/34204 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/37559 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/35865 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/fast/media/invalid-lengths.html:
Which tests are replacing this one and the other deleted one?

iidmsa replied:
They are supposed to be replaced by those WPT tests:
https://wpt.fyi/results/css/mediaqueries?label=master&label=experimental&aligned&q=invalid

I will actually remove them in a separate PR.

iidmsa committed:
Sync `mediaqueries` from WPT upstream
https://bugs.webkit.org/show_bug.cgi?id=294362
rdar://problem/153140956

Reviewed by Anne van Kesteren.

Upstream commit: web-platform-tests/wpt@aae479b

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/display-mode.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-sign-function-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-color-scheme-svg-as-image.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/reference/ref-green-body.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: https://commits.webkit.org/296151@main
